torchvision/datasets/cifar.py

```diff
@@ -46,9 +47,7 @@ class CIFAR10(data.Dataset):
         ['test_batch', '40351d587109b95175f43aff81a1287e'],
     ]
 
-    def __init__(self, root, train=True,
-                 transform=None, target_transform=None,
-                 download=False):
+    def __init__(self, root, train=True, transform=None, target_transform=None, download=False):
```

Comment:
I think this line might be too long and lint is complaining. Can you make it as before?